internal/host/hostutil/host_utils.go

```diff
@@ -360,3 +360,23 @@ func GetIgnitionEndpoint(cluster *common.Cluster, host *models.Host) (string, er
 	}
 	return ignitionEndpointUrl, nil
 }
+
+func GetDisksOfHolderByType(allDisks []*models.Disk, holderDisk *models.Disk, driveTypeFilter models.DriveType) []*models.Disk {
+	disksOfHolder := []*models.Disk{}
+	for _, disk := range allDisks {
+		if disk.DriveType == driveTypeFilter && strings.Contains(disk.Holders, holderDisk.Name) {
+			disksOfHolder = append(disksOfHolder, disk)
+		}
+	}
+	return disksOfHolder
```

Comment:
```suggestion
	disksOfHolder := GetAllDisksOfHolder(allDisks, holderDisk)
	return lo.Filter(disksOfHolder, func (d *models.Disk) { d.DriveType == driveTypeFilter})
```
we can reuse `GetAllDisksOfHolder` here (not sure of the syntax)